kong/meta.lua

```diff
@@ -13,5 +13,14 @@ local version = setmetatable({
 return {
   _NAME = "kong",
   _VERSION = tostring(version),
-  _VERSION_TABLE = version
+  _VERSION_TABLE = version,
+
+  -- third-party dependencies' required version, as they would be specified
+  -- to lua-version's `set()`.
+  _DEPENDENCIES = {
+    nginx = "1.9.15.1",
+    --resty = "", -- not version dependent for now
+    serf  = "0.7.0",
+    --dnsmasq = "" -- not version dependent for now
```

Comment:
I know it does, but we don't have from and to as of now, only a specific version allowed. 

Eventually I would have updated it to be:
```lua
_DEPENDENCIES = {
  nginx = {"1.9.15.1", "1.9.15.2"} -- from - to
}
-- ...
version.set(unpack(meta._DEPENDENCIES.nginx))
```